src/ex1/index.html

```diff
@@ -1,14 +1,32 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <title>Exercise 1</title>
+  <head>
+    <title>Shuki</title>
     <link rel="stylesheet" href="style.css" />
-</head>
-<body>
-
-<h1>Todo list</h1>
-
-
-</body>
-<script src="script.js"></script>
+  </head>
+  <body>
+    <div class="background">
+      <div class="content">
+        <h1>Shuki</h1>
+        <section class="top">
+          <input
+            type="text"
+            class="todoInput"
+            placeholder="Add your new todo"
+          />
+          <button class="addButton" onclick="addTodo()">+</button>
```

Comment:
adding event listeners from the html is considered a bad practice as it assumes a function with that name exists in the context (window in this case) and is unexpected (you can't see in the js code who is calling it)